BUILD.md

```diff
@@ -23,5 +23,15 @@ To find the jar once it's built, just look at the last few lines of output from
 If you're going to make changes to the Scala code, you may want to use IntelliJ and it's Scala
 plugin to help with the Scala syntax...!
 
+If you use [Eclipse IDE](http://www.eclipse.org/), you can set-up your development environment by following these instructions:
+
+* Install `sbt` and build as-above
+* Install [Scala IDE for Eclipse](http://scala-ide.org/) into your Eclipse installation if not already installed
```

Comment:
OK gotcha, done.  I guess I had assumed that this would be late-resolved, and unless/until someone ran `sbt eclipse` it might not impact them, but that may not be the case.  I assume that the trade-off is that locally-defined plugins need to be set-up per-developer-environment, and may need some maintenance as versions of sbt change.

My familiarity with sbt did not extend to centralized/local plugins, but I've adopted that and updated the PR to be only doc changes to `BUILD.md`, plus the `.gitignore` changes.

Interestingly, when I ran `sbt eclipse` having made this change, and I may have done something different, or it may be coincidental, sbt seemed to upgrade itself ... it spent a lot of time fetching a (new?) version `Getting org.scala-sbt sbt 0.13.11 ...`.